lldb/test/API/tools/lldb-dap/attach/TestDAP_attach.py

```diff
@@ -200,7 +200,10 @@ def test_commands(self):
         # Get output from the console. This should contain both the
         # "exitCommands" that were run after the second breakpoint was hit
         # and the "terminateCommands" due to the debugging session ending
-        output = self.collect_console(duration=1.0)
+        output = self.collect_console(
+            timeout_secs=1.0,
```

Comment:
These timeouts look like they're way too small for CI (should be more in the 10s of seconds range and under ASAN they should be multipled by a factor 10).